Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -1021,7 +1021,9 @@ class BrowserViewController: UIViewController,
     /// it's the zero search page, aka when the home page is shown by clicking the url bar from a loaded web page.
     func showEmbeddedHomepage(inline: Bool) {
         guard let isPrivate = browserViewControllerState?.usePrivateHomepage, !isPrivate else {
-            browserDelegate?.showPrivateHomepage()
+            browserDelegate?.showPrivateHomepage(
+                overlayManager: overlayManager
+            )
```

Comment:
```suggestion
            browserDelegate?.showPrivateHomepage(overlayManager: overlayManager)
```
nit: this is pretty simple so it doesn't need to be chopped up